manifests/init.pp

```diff
@@ -301,6 +303,7 @@
   Integer[0] $foreman_service_puma_threads_min = $foreman::params::foreman_service_puma_threads_min,
   Integer[0] $foreman_service_puma_threads_max = $foreman::params::foreman_service_puma_threads_max,
   Integer[0] $foreman_service_puma_workers = $foreman::params::foreman_service_puma_workers,
+  Hash $rails_cache_store = $foreman::params::rails_cache_store,
```

Comment:
```suggestion
  Hash[String, Any] $rails_cache_store = $foreman::params::rails_cache_store,
```

Reply:
Adjusted and pushed, tests are running again